contracts/storage/RibbonTreasuryVaultStorage.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.4;
+
+abstract contract RibbonTreasuryVaultStorageV1 {
+    // Logic contract used to price options
+    address public optionsPremiumPricer;
+    // Logic contract used to select strike prices
+    address public strikeSelection;
+    // Premium discount on options we are selling (thousandths place: 000 - 999)
+    uint256 public premiumDiscount;
+    // Current oToken premium
+    uint256 public currentOtokenPremium;
+    // Last round id at which the strike was manually overridden
+    uint16 public lastStrikeOverrideRound;
+    // Price last overridden strike set to
+    uint256 public overriddenStrikePrice;
+    // Auction duration
+    uint256 public auctionDuration;
+    // Auction id of current option
+    uint256 public optionAuctionID;
+    // Amount locked for scheduled withdrawals last week;
+    uint256 public lastQueuedWithdrawAmount;
+    // Allowed asset for premium denomination
+    mapping(address => bool) allowedAssets;
+    // The asset which denominates the premium during auction
+    address public premiumAsset;
+}
+
+// We are following Compound's method of upgrading new contract implementations
```

Comment:
I see no reason to add this comment

Reply:
We should keep this, just as a reminder that we need to add V2, V3, ... if we need additional storage. Our other vault storages have the same comments too.